python_ta/transforms/type_inference_visitor.py

```diff
@@ -178,10 +181,20 @@ def set_boolop_type_constraints(node):
 # Statements
 ##############################################################################
 def set_assign_type_constraints(node):
-    first_target = node.targets[0]
-    TYPE_CONSTRAINTS.unify(node.frame().type_environment.lookup_in_env(first_target.name),
-                           node.value.type_constraints.type)
-    node.type_constraints = TypeInfo(NoType)
+    # multi-assignment; has "elements"
+    try:
```

Comment:
The `try-except` is not appropriate here. Check whether `node.targets[0]` is an instance of `astroid.Tuple` or not.